### PR TITLE
Email templates updated as per bug 1196306 

### PR DESCRIPTION
**Pull request by satdav**
I understand we \are getting rid of this but I would like Mozilla to do this if possible at present

### CODE REVIEW

**Comment by satdav** [comment]
I cant squeeze them or I would of as I am on windows @flodolo  can you please do this if you have a second just the templates

**Comment by satdav** [comment]
also I know what we are getting rid of these soon the email templates

**Comment by satdav** [comment]
The only one I am not happy about is the education one can you keep that to the same as on the now but merge the rest

File: en-GB/templates/mozorg/emails/education.txt [hunk 1]
```diff
@@ -1,45 +1,14 @@
...
-Thank you for your interest in education and teaching the web. Please feel free to use and remix these resources and to share your experiences with the Webmaker Mentor community.
-
-https://lists.mozilla.org/listinfo/webmaker
-
-All the best,
-
--- The Mozilla Webmaker Mentor team
+Webmaker Mentors
+ 
+Webmaker Mentors are fiercely unconventional technologists, thinkers, and creators. We care about empowering everyone to be makers of the web â and pushing the limits of what's possible online.
```
Comment (flodolo) [inline en-GB/templates/mozorg/emails/education.txt]: What's this `â`?

**Comment by flodolo** [comment]
I'm going to close this PR, since we're going to get rid of emails and autoresponders in the near future (before the end of the month).

**Pull request closed without merging**